doc/classes/Array.xml

```diff
@@ -62,6 +62,13 @@
 				Creates a typed array from the [param base] array.
 			</description>
 		</constructor>
+		<constructor name="Array" qualifiers="typed">
+			<return type="Array" />
+			<param index="0" name="base" type="Array" />
+			<description>
+				Alias for `Array(base, type, class_name, script)`. Implicitly fills type-related parameters from the type provided. Creates a typed array from the [param base] array.
```

Comment:
```suggestion
				Alias for [code]Array(base, type, class_name, script)[/code]. Implicitly fills type-related parameters from the type provided. Creates a typed array from the [param base] array.
```

But see also the `--doctool` output, it doesn't seem to support the two different `Array(Array)` constructors, it might need more code to include the `typed` qualifiers for disambiguation.

Reply:
Fixed, thanks!